src/popup/pages/Settings.tsx

```diff
@@ -1,43 +1,9 @@
 import React from 'react';
-import { ConnectWalletForm } from '@/popup/components/ConnectWalletForm';
 import { WalletInformation } from '@/popup/components/WalletInformation';
-import { useMessage, usePopupState } from '@/popup/lib/context';
-import { getWalletInformation } from '@/shared/helpers';
+import { usePopupState } from '@/popup/lib/context';
 
 export const Component = () => {
   const { state } = usePopupState();
-  const message = useMessage();
 
-  if (state.connected) {
-    return <WalletInformation info={state} />;
-  } else {
-    const connectState = state.transientState['connect'];
-    return (
-      <ConnectWalletForm
-        publicKey={state.publicKey}
-        state={connectState}
-        defaultValues={{
-          recurring:
-            localStorage?.getItem('connect.recurring') === 'true' || false,
-          amount: localStorage?.getItem('connect.amount') || undefined,
-          walletAddressUrl:
-            localStorage?.getItem('connect.walletAddressUrl') || undefined,
-          autoKeyAddConsent:
-            localStorage?.getItem('connect.autoKeyAddConsent') === 'true',
-        }}
-        saveValue={(key, val) => {
-          localStorage?.setItem(`connect.${key}`, val.toString());
-        }}
-        getWalletInfo={getWalletInformation}
-        connectWallet={(data) => message.send('CONNECT_WALLET', data)}
-        onConnect={() => {
-          // The popup closes due to redirects on connect, so we don't need to
-          // update any state manually.
-          // But we reload it, as it's open all-time when running E2E tests
-          window.location.reload();
-        }}
-        clearConnectState={() => message.send('CONNECT_WALLET', null)}
-      />
-    );
-  }
+  return <WalletInformation info={state} />;
```

Comment:
Should we move the implementation for `WalletInformation` here in `pages/Settings.tsx` directly? It feels a bit redundant if we only return a component.

Reply:
I'm going this way after this PR: https://github.com/interledger/web-monetization-extension/pull/657, in case this is what you're saying.